src/Storefront/Controller/CartLineItemController.php

```diff
@@ -70,6 +70,14 @@ public function deleteLineItem(Cart $cart, string $id, Request $request, SalesCh
         });
     }
 
+    #[Route(path: '/checkout/cart/delete', name: 'frontend.checkout.cart.delete', defaults: ['XmlHttpRequest' => true], methods: ['POST'])]
+    public function deleteCart(Request $request, SalesChannelContext $context): Response
+    {
+        $this->cartService->deleteCart($context);
```

Comment:
we should call the route here, as you can see, it executes some additional logic, which we might also consider